cogboard-app/src/main/kotlin/com/cognifide/cogboard/widget/type/JiraBucketWidget.kt

```diff
@@ -0,0 +1,58 @@
+package com.cognifide.cogboard.widget.type
+
+import com.cognifide.cogboard.widget.AsyncWidget
+import io.vertx.core.Vertx
+import io.vertx.core.json.JsonArray
+import io.vertx.core.json.JsonObject
+import com.cognifide.cogboard.CogboardConstants as CC
+
+class JiraBucketWidget(vertx: Vertx, config: JsonObject) : AsyncWidget(vertx, config) {
+
+    private val bucketQueries: JsonArray = config.getJsonArray("bucketQueries")
+
+    private val buckets: JsonArray = prepareBuckets()
+
+    private fun prepareBuckets(): JsonArray {
```

Comment:
instead of specifying this method use `init` block

Reply:
Done, thanks.

Reply:
where?

Reply:
in 5e9823c389f17db7ef25c5707c73438bb2f11956